Partition (WBP)/Files/VTLI0036A.DBO.file.yml

```diff
@@ -27,19 +27,19 @@
 
   Multilingual Details:
   - DAN:
-      Description:        "VTLI0036A"
+      Description:        "VTLI0036A in MSSQLS"
```

Comment:
The English needs to be changed too.

Reply:
![image](https://github.com/lansa/db-regression/assets/122512451/ced3c43e-77a6-4990-8a7b-fad45971f753)

For all tables English description goes to default name as soon as we change and save them.